static/src/js/zustand/types.ts

```diff
@@ -866,6 +866,34 @@ export type UiSlice = {
   }
 }
 
+// Holds project panel state this is: panel.group.section
+// Example 0.5.2 is the 5th collection in the project and looking at the panel variable details
+// 1.3.0 is the granule list for the third collection in the project
+export type ActivePanelConfiguration =`${number}.${number}.${number}`
```

Comment:
Could this be expanded to better describe what each number is? 

If something like this works, you can be more specific about the param type of setPanelGroup and setPanelSection

```
type PanelsPanel = `${number}`
type PanelsGroup = `${number}`
type PanelsSection = `${number}`
export type ActivePanelConfiguration = `${PanelsPanel}.${PanelsGroup}.${PanelsSection}`
```
Then this for the function params
```
setPanelGroup: (group: PanelsGroup) => void 
```